### PR TITLE
define appointment event

### PR DESCRIPTION
**Pull request by lantrannk**

### CODE REVIEW

File: lib/features/appointments/bloc/appointment_event.dart [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import 'package:salon_appointment/features/appointments/model/appointment.dart';
+
+abstract class AppointmentEvent {
+  const AppointmentEvent();
+}
+
+class FetchAppointment extends AppointmentEvent {}
```
Comment (nhandoanngoc-agilityio) [inline lib/features/appointments/bloc/appointment_event.dart]: https://bloclibrary.dev/#/blocnamingconventions please refer the link and update the naming conventions

File: lib/features/appointments/bloc/appointment_event.dart [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import 'package:salon_appointment/features/appointments/model/appointment.dart';
+
+abstract class AppointmentEvent {
+  const AppointmentEvent();
+}
+
+class AppointmenFetch extends AppointmentEvent {}
+
+class AppointmentLoad extends AppointmentEvent {}
```
Comment (ningo-agilityio) [inline lib/features/appointments/bloc/appointment_event.dart]: AppointmentLoading

File: lib/features/appointments/bloc/appointment_event.dart [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import 'package:salon_appointment/features/appointments/model/appointment.dart';
+
+abstract class AppointmentEvent {
+  const AppointmentEvent();
+}
+
+class AppointmenFetch extends AppointmentEvent {}
```
Comment (ningo-agilityio) [inline lib/features/appointments/bloc/appointment_event.dart]: typo mistake

File: lib/features/appointments/bloc/appointment_event.dart [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+class AppointmentLoad extends AppointmentEvent {}
+
+class AppointmentAdd extends AppointmentEvent {}
```
Comment (ningo-agilityio) [inline lib/features/appointments/bloc/appointment_event.dart]: What is payload data? Did you see request data in AppointmentEdit?

**Pull request closed without merging**